src/HackF5.UnitySpy/Detail/ProcessFacade.cs

```diff
@@ -105,8 +132,11 @@ public object ReadManaged([NotNull] TypeInfo type, uint address)
                 case TypeCode.OBJECT:
                     return this.ReadManagedGenericObject(type, address);
 
-                // may need supporting
                 case TypeCode.VAR:
+                    // Really not sure this is the way to do it
+                    return this.ReadInt32(address);
```

Comment:
FYI this doesn't work in all cases, and will need proper support at some point

Reply:
Yeah this is the main issue why dictionaries are not working since the fields of the Entry nested struct are VAR type coded.